src/quo2/components/calendar/calendar_month/utils.cljs

```diff
@@ -0,0 +1,23 @@
+(ns quo2.components.calendar.calendar-month.utils
+  (:require [utils.i18n :as i18n]
+            [utils.datetime :as dt]))
+
+(defn format-month-year
+  [year month]
+  (let [month (cond
+                (or (nil? month) (zero? month)) 1
+                (> month 12)                    12
+                :else                           month)]
+    (str (i18n/label (get dt/full-months month)) " " year)))
```

Comment:
This should be done by
'(i18n/label (get dropped/full-months month)) {:year year})'

Because I think this structure can depend on different languages so better if it is an argument of the string- there are some examples of this in the en.json file if you look for strings using '{' or '}'

Reply:
If you add the year to the months in translations ("january": "January {{year}}"), it might limit their usability in other places. Consider the specific use case and decide accordingly. Shall I proceed with the change?

Reply:
Well in other places in the app using month that would be a different translation string. it's not really about reusability across the codebase when it comes to these translation strings. Thanks for confirming 👍

Reply:
Maybe make the key a bit more appropriate for the use case 👍

Reply:
I resolved it and used a proper key for translation